src/util.rs

```diff
@@ -6,7 +6,7 @@ use log::warn;
 use std::{error::Error, fs};
 use toml_edit::{Document, InlineTable, Item, Table, Value};
 
-pub fn members_deep<'a>(ws: &'a Workspace) -> Vec<Package> {
+pub fn members_deep(ws: &'_ Workspace) -> Vec<Package> {
```

Comment:
Is that machine applied? It looks like regular `(ws: &Workspace)` would work here